### PR TITLE
PLENTY-337 add configuration for showing of selected filters count

### PR DESCRIPTION
**Pull request by RimantasAdeoWeb**

### CODE REVIEW

File: resources/views/ItemList/Components/Filter/ItemFilter.twig [hunk 1]
```diff
@@ -12,7 +12,7 @@
             <item-range-slider template-override="#vue-findologic-item-range-slider" :facet="facet"></item-range-slider>
         </div>
         <div v-else-if="facetType === 'image'">
-            <item-filter-image template-override="#vue-findologic-item-filter-image" :facet="facet" :fallbackImage="'{{ plugin_path("Findologic") }}/images/no-picture-2.png'"></item-filter-image>
+            <item-filter-image template-override="#vue-`findologic-item-filter-image" :facet="facet" :fallbackImage="'{{ plugin_path("Findologic") }}/images/no-picture-2.png'"></item-filter-image>
```
Comment (TheKeymaster) [inline resources/views/ItemList/Components/Filter/ItemFilter.twig:15]: What is this change for?

File: resources/lang/de/Config.properties [hunk 2]
```diff
@@ -12,6 +12,9 @@ filtersStylingFilterPerRow1=1
 filtersStylingFilterPerRow2=2
 filtersStylingFilterPerRow3=3
 filtersStylingFilterPerRow4=4
+filtersShowSelectedFiltersCountLabel=Show selected filters count
+filtersShowSelectedFiltersDisabled=No
+filtersShowSelectedFiltersEnabled=Yes
```
Comment (TheKeymaster) [inline resources/lang/de/Config.properties:17]: That's not German.

**Review by TheKeymaster — CHANGES_REQUESTED** [review]

File: resources/lang/de/Config.properties (hunk 2 above)
Comment (RimantasAdeoWeb) [reply resources/lang/de/Config.properties:17]: I don't have the German translations. If you add them here, I'll update these.

Comment (RimantasAdeoWeb) [reply resources/lang/de/Config.properties:17]: Ah, sorry, they were in the story after all. Missed them at first.

**Review by TheKeymaster — APPROVED** [review]

**Pull request merged**